### PR TITLE
Created class Person and their inheritance, Student and Teacher

### PR DESCRIPTION
**Pull request by stanleySimeon**
# In this milestone

  - **I created the class Person and their inheritance (Student and Teacher)**
      - created getters and setters for `@id` `@name` and `@age`
      - created a private and public method
      
    **_The class Student inherited from the class Person and has_**
        - a constructor extends the parent's constructor by adding @classroom and a parameter for it.
        - a Method play_hooky that returns "¯\(ツ)/¯".
     
    **_The class Teacher inherited from the class Person and has_**
        - a Constructor extends the parent's constructor by adding @specialization and a parameter for it.
        - an override can_use_services? to always returns true.

### CODE REVIEW

File: person.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Create a class person
```
Comment (arslanbisharat) [inline person.rb:1]:
```ruby

- [ ] instance variables should be able to be modified that's why they are defined as attr_accessors.

  attr_accessor :name, :age 
  

- [ ]  *:id should only be able to be retrieved so only the Getter is defined

  attr_reader :id 
```

Comment (arslanbisharat) [inline person.rb:1]: - [ ] :name and :age instance variable should be Getters and Setters are defined
  attr_accessor :name, :age 

- [ ]  :id should only be able to be retrieved so only the Getter is defined

  attr_reader :id

File: person.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        @id = id
+        @name = name
+        @age = age
+    end
+    def initialize(name, age, parent_permission = true)
+        @name = name
+        @age = age
+        @parent_permission = parent_permission
+    end
+    # create a getters for @id, @name, and @age
```
Comment (arslanbisharat) [inline person.rb]: - [x] Please make sure you don't have two methods with same name.

File: .vscode/settings.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (arslanbisharat) [inline .vscode/settings.json]: - [x] Please remove .vscode from your project.

File: .vscode/settings.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (arslanbisharat) [inline .vscode/settings.json]: - [x] Please make your project readme more descriptive.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
Hi @stanleySimeon, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- The Pull Request has a proper title and description. ✔️
- You haven't added a descriptive Readme file. :x:
- There is no linter error is present. ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ] Please make sure you right descriptive comments. Commenting involves placing Human Readable Descriptions inside of computer programs detailing what the Code is doing. Proper use of commenting can make code maintenance much easier, as well as helping make finding bugs faster. Further, commenting is very important when writing functions that other people will use. :x:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: person.rb (hunk 1 above)
Comment (stanleySimeon) [reply person.rb:1]: Hello! @arslanbisharat, Kindly did you read the project requirements?

File: person.rb (hunk 2 above)
Comment (stanleySimeon) [reply person.rb]: I follow the project requirements can you check it again, please!?

File: .vscode/settings.json (hunk 4 above)
Comment (stanleySimeon) [reply .vscode/settings.json]: Thanks
I'm forget that!

File: .vscode/settings.json (hunk 3 above)
Comment (stanleySimeon) [reply .vscode/settings.json]: thanks

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @stanleySimeon, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by stanleySimeon** [comment]
Thanks for your help my bro, I missed something for the first request!

**Pull request merged**